integration/test.ats

```diff
@@ -113,43 +124,86 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
     # _serac_tolerance_test_end
```

Comment:
Add documentation for `restart_cycle` here which will get included in the sphinx docs.